addons/mail/static/src/core/common/notification_model.js

```diff
@@ -56,6 +55,24 @@ export class Notification extends Record {
     });
     /** @type {string} */
     failure_type;
+    get failureMessage() {
```

Comment:
I suppose we also have to update in SMS (and to check in snailmail). Whatsapp has its own bidouille I think. But at least SMS and I think snailmail (to check) have the same kind of behavior than email notifications.